roles/os_firewall/defaults/main.yml

```diff
@@ -1,9 +1,5 @@
 ---
 os_firewall_enabled: True
-# TODO: Upstream kubernetes only supports iptables currently
-# TODO: it might be possible to still use firewalld if we wire up the created
-# chains with the public zone (or the zone associated with the correct
-# interfaces)
-os_firewall_use_firewalld: False
+os_firewall_use_firewalld: True
```

Comment:
Why not set this to `"{{ False if openshift.common.is_atomic | bool else True }}"

Then, you could replace the set_fact in tasks/main.yml with:
```
- assert: 
    that:
    - "{{ openshift.common.is_atomic | bool is true and os_firewall_use_firewalld | bool is true }}"
```
instead of using set_fact